This commit:
https://chromium.googlesource.com/chromium/src/+/224e825bba5d74418ac2d38cf2bea004dcd698ec%5E%21/

...adds an empty isHangulArray[], which breaks ICU:

../../third_party/blink/renderer/platform/text/character.cc: In static member function ?static bool blink::Character::IsHangulSlow(UChar32)?:
../../base/macros.h:57:55: error: no matching function for call to ?ArraySizeHelper(const UChar32 [0])?

This patch is taken from Arch Linux:
https://git.archlinux.org/svntogit/packages.git/tree/trunk/chromium-system-icu.patch?h=packages/chromium

diff --git a/third_party/blink/renderer/platform/text/character_property_data_generator.h b/third_party/blink/renderer/platform/text/character_property_data_generator.h
index 28fb6a9..bb4dbd7 100644
--- a/third_party/blink/renderer/platform/text/character_property_data_generator.h
+++ b/third_party/blink/renderer/platform/text/character_property_data_generator.h
@@ -244,10 +244,12 @@ static const UChar32 kIsHangulRanges[] = {
     0xD7B0, 0xD7FF,
     // Halfwidth Hangul Jamo
     // https://www.unicode.org/charts/nameslist/c_FF00.html
-    0xFFA0, 0xFFDC,
+    0xFFA0, 0xFFDB,
 };
 
-static const UChar32 kIsHangulArray[] = {};
+static const UChar32 kIsHangulArray[] = {
+  0xFFDC,
+};
 
 }  // namespace blink
 
